bin/node-template/flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }: flake-utils.lib.eachDefaultSystem (system:
+    let
+      pkgs = import nixpkgs { inherit system; };
+    in
+    {
+      devShells.default = pkgs.mkShell {
+        packages = with pkgs; [
+          rustup
+          clang
+          protobuf
+          rustfmt
```

Comment:
You install here `rustfmt` and then also via the `rust-toolchain.toml`. One should be removed.

Reply:
rustup seems to ignore `rust-toolchain.toml's` furstfmt at times. Seems to be working now so I cleaned it up.